### ISSUE TITLE
Enlarge 'text boxes' on grid

### ISSUE THREAD

**Issue by trysteroids** [aproxima/htmlq#7]
Hi all,

I'm having a problem with HTMLq when using many statements.  I am on a research team and am responsible for configuring our q surveys. 

My research project calls for 46 statements, all quite long.  In Step 2 after I place statements onto the grid, I unfortunately cannot read anything, even after I use smaller fonts.  I anticipate that my survey participants will become frustrated when they're unable to read the text once they place it onto the grid.

Basically I do not want the program to compress the font size when I increase the array dimensions (we are using enough squares to cover 46 responses, ranging -5 to +5). 

I'm not sure which file to edit directly to change this display. I suspect it's htmlq.js but I don't know Javascript really and am at a loss on what to do.  

Any help is appreciated, thanks.
